MetaMask/metamask-extension
# Adding shadow docs to storybook

## Explanation

Adding documentation regarding shadow and the use of design token css variables to storybook

## More Information

* Fixes #15299 

## Screenshots/Screencaps

### After

![screencapture-localhost-6007-2022-07-29-11_49_51](https://user-images.githubusercontent.com/8112138/181825823-fde86b5e-55d1-472d-a468-50d7e87e70c7.png)


## Manual Testing Steps
- Go to the storybook build of this PR (Will need to have been rebased after this [PR](https://github.com/MetaMask/metamask-extension/pull/15387) is merged 
- Go to Foundations / Shadow in the side navigation
- Read the shadow docs and check if they make sense 

## Pre-Merge Checklist


### + If there are functional changes:
